libkbfs/folder_branch_ops.go

```diff
@@ -4191,6 +4191,18 @@ func (fbo *folderBranchOps) syncAllUnlocked(
 	ctx context.Context, lState *lockState) error {
 	fbo.mdWriterLock.Lock(lState)
 	defer fbo.mdWriterLock.Unlock(lState)
+
+	select {
+	case <-ctx.Done():
```

Comment:
What if we get canceled after this line?

Reply:
`ForceCancel` is called under `mdWriterLock`, so we don't have to worry about that.